src/plugins/discover/public/plugin.tsx

```diff
@@ -263,7 +261,7 @@ export class DiscoverPlugin
         defaultMessage: 'JSON',
       }),
       order: 20,
-      component: ({ hit, indexPattern }) => (
+      Component: ({ hit, indexPattern }) => (
```

Comment:
I think we should revert the change from `component` to `Component`. Why? We provide a registry for 3rd party doc views, so every additional registration would need to apply this change. I agree that `Component` is a better fit here, but for backwards compatibility it should be `component`. )